### DISCUSSION TITLE
Integration with orestbida/cookieconsent

### DISCUSSION THREAD

**Discussion by jelen07** [orestbida/iframemanager#22] [Ideas]
Hi, is there any way how to integrate those two tools together?

### UC
1. User add positive consent with category, which include YouTube videos.
2. There is no need to requride additional consent for iframemanager, because it was already given by cookieconsent.

and vice versa.

**Comment by mr-manuel** [comment]
It's already possible for cookieconsent to iframemanager. You find an example here: https://github.com/orestbida/cookieconsent/tree/master/demo/demo_iframemanager

For vice versa it should also be possible in a similar way.
